pkg/validation/apiextensions/v1/composition/validator.go

```diff
@@ -133,6 +133,7 @@ func (v *Validator) Validate(ctx context.Context, obj runtime.Object) (warns []s
 		v.validatePatchesWithSchemas,
 		v.validateReadinessChecksWithSchemas,
 		v.validateConnectionDetailsWithSchemas,
+		v.validateEnvironmentPatchesWithSchemas,
 		// TODO(phisco): add more phase 2 validation here
```

Comment:
do we still need this comment, or we want to just keep it as a placeholder?

Reply:
I was thinking to leave it there just as a placeholder for future reference